Admin page mark/unmark mentors

Lbatson opened the pull request:
implemented feature from #4 to select and deselect mentors from admin pages. checks if user is admin before allowing access. insures mentor wont remove themselves. updates all users is_mentor field based on selection when saved.

Lbatson committed:
allow adding and removing mentors from admin page.

raylu commented:
1. this is pretty messy, please squash some commits
2. does this merge onto master (with the merged css changes?)

Lbatson commented:
alright, commits squashed. fixed the multi line to single line quotes. removed the css changes i made that weren't part of the admin page. I'll get those sorted out when i go back and edit the theme again.

raylu commented on templates/admin.html:
```diff
@@ -0,0 +1,36 @@
+		<div class="col-md-12">
+			<form action="/admin" method="post">
+				<input class="btn btn-success pull-right" type="submit" value="Save Changes">
+				<br><br><br>
```
:(

Lbatson commented:
heres a gif of the feature currently

![test](https://cloud.githubusercontent.com/assets/1441464/2738969/90828406-c69d-11e3-9780-310cb230c0ec.gif)

raylu commented on server.py:
```diff
@@ -216,6 +216,35 @@ def post(self):
+				self.db.update_is_mentor(remove_mentor_set, 0)
+			self.redirect('/admin')
+		else:
+			self.redirect('/')
```
this should definitely be `raise tornado.web.HTTPError(403)`

the other auth-ed handlers check auth, raise if conditions aren't met, and everything else is unindented

Lbatson replied:
updated this

raylu commented on static/css/base.ccss:
```diff
@@ -87,6 +87,19 @@ body:
 		right: 0
 		width: 85%
+
+/* admin.html */
```
:(

Lbatson replied:
removed

raylu commented:
wait, i totally didn't realize we already had a bunch of page-specific css in the same base.ccss file...

the :( was because we already have page-specific css in separate files

Lbatson commented:
ok i'll split out the page sepecific stuff to their own files

Lbatson committed:
fix and split out css files for pages.
